src/util/debug.h

```diff
@@ -31,6 +31,11 @@
 #define SSS_ATTRIBUTE_PRINTF(a1, a2)
 #endif
 
+#ifndef HAVE_ERRNO_T
+#define HAVE_ERRNO_T
+typedef int errno_t;
+#endif
+
```

Comment:
Why do we need another place in src/util that defines errno? Wouldn't it be better to include the existing header before debug.h (I know our headers under util/ are a mess..).